ssh/tailssh/tailssh.go

```diff
@@ -1004,14 +1005,20 @@ func (ss *sshSession) run() {
 	go ss.killProcessOnContextDone()
 
 	go func() {
+		defer ss.stdin.Close()
 		_, err := io.Copy(rec.writer("i", ss.stdin), ss)
 		if err != nil {
 			// TODO: don't log in the success case.
 			logf("stdin copy: %v", err)
 		}
-		ss.stdin.Close()
+		if f, ok := ss.stdin.(*os.File); ok {
+			unix.IoctlSetInt(int(f.Fd()), unix.TIOCSBRK, 0)
```

Comment:
If we later import https://github.com/pkg/term we can use https://github.com/pkg/term/blob/master/termios/termios_linux.go#L35 which also has fall-back implementations for bsd's, solaris, etc.

Reply:
Yeah that's a good find. I created a track bug to investigate that.